openslides_backend/models/base.py

```diff
@@ -124,3 +124,17 @@ def get_properties(self, *fields: str) -> Dict[str, fields.Schema]:
         for field in fields:
             properties.update(self.get_property(field))
         return properties
+
+    def get_required_fields(self) -> Iterable[fields.Field]:
+        """
+        Yields all required fields
+        """
+        for model_field in self.get_fields():
+            if model_field.required and isinstance(model_field, fields.Field):
```

Comment:
What is the use of the `isinstance` check here? Shouldn't this always be a `Field`?

Reply:
Removed! I remember there was an issue, but now all tests are successful without the isinstance.